instat/dlgOneWayFrequencies.vb

```diff
@@ -170,8 +152,8 @@ Public Class dlgOneWayFrequencies
     End Sub
 
     Private Sub ucrBase_BeforeClickOk(sender As Object, e As EventArgs) Handles ucrBase.BeforeClickOk
-        If rdoTable.Checked Then
-            ucrBase.clsRsyntax.SetBaseROperator(clsTableBaseOperator)
+        If rdoTable.Checked OrElse rdoBoth.Checked Then

```

Comment:
I get you right. It runs both table and plot correctly on first run, but behaves strangely on second and subsequent runs before you reset (only runs tables) checking on it though.